### DISCUSSION TITLE
Bumping minimum jre to 11

### DISCUSSION THREAD

**Discussion by sameb** [google/guice#1743] [Announcements]
Hi folks,

As of https://github.com/google/guice/commit/c58bc33b9005978880249b796c1dd593d9fb13ef, Guice will require Java 11 (up from Java 8). This will apply to all future releases (past 7.0.0).

- sam

**Comment by GedMarc** [comment]
Definitely JDK 17 should be considered yes, 
Maybe full compatibility with the inclusion of a module-info file so we can build JLink native builds

**Comment by wendigo** [comment]
@sameb I'd opt for 17 for future versions

**Comment by sameb** [comment]
17 is an interesting question, it's just not the one I'm ready to ask about.

**Comment by GedMarc** [comment]
I was working with the idea that JDK 11 is EOL in September this year :) I understand but seems a bit of a waste of effort?

It's really about being able to build native Java applications without a damn fat-jar and using the modular pathways instead of the "classpath" ;) 
incredible speeds and benefits when you build up a JRE using JLink - also some strange things too (the JRT file system)

**Comment by sameb** [comment]
FYI, the minimum is now bumped to Java 11 (as of https://github.com/google/guice/commit/c58bc33b9005978880249b796c1dd593d9fb13ef).
